### ISSUE TITLE
Error: no storage locations defined in configuration

### ISSUE THREAD

**Issue by bodypilot** [jeffaco/duplicacy-util#27]
Hi,

This might be related to https://github.com/jeffaco/duplicacy-util/issues/19 although I'm not a developer so I'm not sure.

On Windows 10 I run into the following problem when running duplicacy-util:

> C:\Users\user>duplicacy-util -f Laptop-MC -a

```
20:38:53 Using global config: C:\Users\user\.duplicacy-util\duplicacy-util.yaml
Error: no storage locations defined in configuration
Error: no prune locations defined in configuration
Error: no check locations defined in configuration
```
Here's some extra info:

> duplicacy-util -version

`Version: 1.4, Git Hash: cad0ac7`

> C:\Users\user\\.duplicacy-util\Laptop-MC.yaml

```
repository: C:\Users\user

storage:
    -   name: rp3-32

prune:
    -   storage: rp3-32
        keep: "0:365 30:180 7:30 1:7"

check:
    -   storage: rp3-32
```
> C:\Users\user\\.duplicacy\preferences

```
[
    {
        "name": "rp3-32",
        "id": "Laptop-MC",
        "repository": "",
        "storage": "sftp://user@host//mnt/data/backup/Laptop-MC",
        "encrypted": true,
        "no_backup": false,
        "no_restore": false,
        "no_save_password": false,
        "nobackup_file": "",
        "keys": null
    }
]
```

A similar setup on my Linux laptop using the old scheme with numbers works well.

```
repository: /home/user

storage:
           1:
                          name: rp3-32

prune:
           1:
                         storage: rp3-32
                         keep: "0:365 30:180 7:30 1:7"

check:
            1:
                       storage: rp3-32
```
But  if I set this up under Windows 10 it doesn't work.
Am I doing something wrong or have I stumbled upon a bug?

**Comment by jeffaco** [comment]
That is related to #19, so good call! Unfortunately, I thought about this a few weeks ago, and I don't have a stellar solution.

The `README.md` file is updated as things get added to mainline. But you're running `v1.4`, the last released version, and that doesn't include #19. So if you're a newcomer and just look at `README.md`, you're liable to be confused.

Two possible solutions:

1. Release `v1.5` from mainline, and that will work for you, or
2. "Downgrade" your `Laptop-MC.yaml` file to be of the old format (with numbers, like on your Linux system).

If you were using your Linux system to run `duplicacy-util`, the control files from Linux should be identical (just watch line endings because Windows is weird with that).

If "downgrading" your YAML file is difficult for me, let me know and I'll release `v1.5`. But the downgrade should be pretty trivial, and immediately gets you going.

Sorry for your problem. The only way I see to work around this is to keep all `README.md` file changes in a branch until we release the next version, and that's a little sloppy. If I have to do that, I will, but I'd rather not. Of course, NOT making incompatible changes is better, but hopefully that's behind us now.

Let me know if this takes care of you, thanks.

**Comment by jeffaco** [comment]
I went ahead and released `v1.5`, which will resolve this issue. Feel free to run `v1.5`, and that will support your YAML file. Sorry that you encountered this.
